Objects/bytesobject.c

```diff
@@ -3557,4 +3557,3 @@ _PyBytes_Repeat(char* dest, Py_ssize_t len_dest,
         }
     }
 }
-
```

Comment:
I don't like trailing blankline (`\n\nEOF`) too.
But please don't touch this file in this PR.